database/query/projects.sql

```diff
@@ -101,4 +101,8 @@ WHERE id = $1 RETURNING *;
 -- name: UpdateProjectMeta :one
 UPDATE projects
 SET metadata = $2
-WHERE id = $1 RETURNING *;
\ No newline at end of file
+WHERE id = $1 RETURNING *;
+
+-- name: ListAllParentProjects :many
+SELECT * FROM projects
+WHERE parent_id IS NULL AND is_organization = FALSE;
```

Comment:
The `is_organization` boolean is now unused. We should probably just get rid of this column.

Reply:
I'll file an issue so we fix this separately 👍

Reply:
https://github.com/mindersec/minder/issues/4853

Reply:
I'd rename this to `ListRootProjects` to avoid confusion. Since a parent project may itself have a parent.